lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,300 @@
+---
+name: Financial Transaction
+field_name_base_url: http://hl7.org/fhir/r4/Basic
```

Comment:
all the valid links to hl7.org need to be https

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/67f9028a3772bdb85b43fd64dc18e900d47649d0#diff-5125c86cde87f8439cb9d1bb859194e9R15

Reply:
to be clear the hl7.org systems still need to be http:// only the proprietary systems need to be https://

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/1da195dca470524ec6880af1b214b75d737ecabe#diff-26870e1b94fc9955e130f3f940dbe2e7L16